bin/dry-run.rb

```diff
@@ -168,16 +221,35 @@ def show_diff(original_file, updated_file)
   puts "    ~~~"
 end
 
+def get_updated_deps(checker)
```

Comment:
Unrelated to this PR, but I wonder if we can hide this complexity from users of the library by using some sensible defaults.